requirements.txt

```diff
@@ -1,3 +1,4 @@
+backports.csv==1.0.5
```

Comment:
Actually, can we make this a more abstract version specifier?

Reply:
Changed to `>=` with the same version number.